src/utils/latest-version.js

```diff
@@ -0,0 +1,26 @@
+const packageJson = require('package-json')
+const { info } = require('./logger')
+
+module.exports = async function (currentVersion) {
+  try {
+    const { version } = await packageJson('@restqa/restqa')
+    const result = (version === currentVersion)
+    if (result === false) {
+      const message = `
+ --
```

Comment:
This section need to be proof read because it will be exposed to the users

Reply:
Done @olivierodo